gobusterdir/gobusterdir.go

```diff
@@ -112,7 +114,11 @@ func (d GobusterDir) ResultToString(g *libgobuster.Gobuster, r *libgobuster.Resu
 		}
 
 		if !g.Opts.NoStatus {
-			if _, err := fmt.Fprintf(buf, " (Status: %d)", r.Status); err != nil {
+			statusWithColor := Sprintf(Brown(r.Status))
+			if r.Status == 200 {
+				statusWithColor = Sprintf(Green(r.Status))
```

Comment:
I think it's unnecessary to have different colours specifically for 200. If you get a hit, based on the requested status codes, that's success and should be green.